config/routes.rb

```diff
@@ -2,9 +2,11 @@
   scope "(:locale)", locale: /en|vi/ do
     get "/login", to: "sessions#new"
     get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
     get "/about", to: "static_pages#about"
     get "/contact", to: "static_pages#contact"
     get "/service", to: "static_pages#service"
     root "static_pages#home"
+    resources :users, only: %i(show update edit)
```

Comment:
I can see, you also declared [new](https://github.com/awesome-academy/r-oe13-chatroom/pull/8/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R8) action